tests/product/test_installer.py

```diff
@@ -42,30 +42,12 @@ def tearDown(self):
         self.centos_container.tear_down()
 
     @attr('smoketest')
-    def test_online_installer(self):
-        self.pa_installer._build_installer_in_docker(self.centos_container,
-                                                     online_installer=True,
-                                                     unique=True)
+    def test_installer(self):
         self.__verify_third_party_dir(False)
         self.pa_installer.install(
             dist_dir=self.centos_container.get_dist_dir(unique=True))
         self.run_prestoadmin('--help', raise_error=True)
 
-    @attr('smoketest', 'offline_installer')
-    def test_offline_installer(self):
```

Comment:
I wanted to make tests agnostic about what version (offline vs online) they tests.
My idea of having tests for online and offline was to run something like:

```
make dist-online
make test-all
```
and 
```
make dist-offline
make test-all
```

Now I noticed that `test_offline_installer` disables network in the container. I will revert it. But I remove it from the group `smoketests`, so that test should be run only with offline installer.